Sources/Testing/Testing.docc/MigratingFromXCTest.md

```diff
@@ -355,6 +355,11 @@ their equivalents in the testing library:
 | `try XCTUnwrap(x)` | `try #require(x)` |
 | `XCTFail("…")` | `Issue.record("…")` |
 
+- Note: When comparing two numeric values within a specified accuracy, i.e. the
+analogue of [`XCTAssertEqual(_:_:accuracy:_:file:line:)`](https://developer.apple.com/documentation/xctest/3551607-xctassertequal)
+, please import [the swift-numerics package](https://github.com/apple/swift-numerics)
```

Comment:
```suggestion
, import [the swift-numerics package](https://github.com/apple/swift-numerics)
```

Reply:
I'd also reduce the link to just "swift-numerics" but that's not a hard rule, just my preference.